metrics/tarantool/luajit.lua

```diff
@@ -1,30 +1,9 @@
-local metrics = require('metrics')
```

Comment:
I hope you considered this thread when push this changes https://github.com/tarantool/metrics/pull/128#discussion_r505303306